pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -269,6 +272,12 @@ const (
 	NodeGroupTypeUnmanaged NodeGroupType = "unmanaged"
 	// NodeGroupTypeUnowned defines an unowned managed nodegroup
 	NodeGroupTypeUnowned NodeGroupType = "unowned"
+	// DefaultNodeVolumeThroughput defines the default throughput for gp3 volumes
+	DefaultNodeVolumeThroughput = 125
+	// DefaultNodeVolumeIO1IOPS defines the default throughput for gp3 and io1 volumes
+	DefaultNodeVolumeIO1IOPS = 100
+	// DefaultNodeVolumeGP3IOPS defines the default throughput for gp3 and io1 volumes
+	DefaultNodeVolumeGP3IOPS = 3000
```

Comment:
Are these defaults from an AWS doc? Could we link it in the comment and/or in the PR?